docs/getting-started.md

```diff
@@ -7,28 +7,126 @@ title: Getting Started
+|-----                              |------                         |
+|wozobjecten                        |identificatie, _links.self     |
+
+- [HAL links](https://tools.ietf.org/html/draft-kelly-json-hal-08), die soms [templated](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.2.0/features/uri-templating.feature) worden geleverd.
 
 ## Bouw de API
 We hebben [client code](https://github.com/VNG-Realisatie/Haal-Centraal-WOZ-bevragen/tree/master/code){:target="_blank" rel="noopener"} voor API-clients in enkele varianten. Hiermee kan je direct aan de slag met het gebruiken van de API.
 
-Zit jouw gewenste ontwikkelomgeving er niet bij, dan kan je zelf ook code genereren vanuit de "[genereervariant](https://github.com/VNG-Realisatie/Haal-Centraal-WOZ-bevragen/tree/master/specificatie/genereervariant/openapi.yaml){:target="_blank" rel="noopener"}" van de API-specificaties.
+Zit jouw gewenste ontwikkelomgeving er niet bij, dan kan je zelf ook code genereren vanuit de "[genereervariant](https://github.com/VNG-Realisatie/Haal-Centraal-WOZ-bevragen/blob/master/specificatie/genereervariant/openapi.yaml){:target="_blank" rel="noopener"}" van de API-specificaties.
 
 ## Probeer en test de API
-Om te testen heb je een API-key nodig. Deze plaats je in requests in de header "x-api-key".
+De werking van de API is het makkelijkst te testen met behulp van [Postman](https://www.getpostman.com/).
+We hebben al een project voor je gemaakt die je kan gebruiken: [WOZ-Bevragen-postman-collection.json](../test/WOZ-Bevragen-postman-collection.json). Hierin moet je alleen de endpoints en authenticatie (API-key) nog invullen.
+We hebben al een [Postman collection](https://github.com/VNG-Realisatie/Haal-Centraal-WOZ-bevragen/blob/master/test/WOZ-Bevragen-postman-collection.json){:target="_blank" rel="noopener"} voor je klaargezet. Deze kun je importeren in Postman.
+
+### Configureer de url en api key
+
+1. Klik bij "Waardering onroerende zaken" op de drie bolletjes.
+2. Klik vervolgens op Edit
+3. Selecteer tabblad "Authorization"
+4. Kies TYPE "API Key"
+5. Vul in Key: "x-api-key", Value: de API key die je van het Kadaster hebt ontvangen, Add to: "Header"
+6. Selecteer tabblad "Variables"
+7. Vul bij baseUrl INITIAL VALUE en bij CURRENT VALUE de url
+8. Klik op de knop Update
+
+De testomgeving van de API is te benaderen via de volgende urls:
+- _Beveiligde verbinding met alleen API-key: https://api.kadaster.nl/lvwoz-eto/huidigebevragingen_
+    - Voor deze connectie met de testomgeving van deze API is vereist:
+        - Een geldige API-key. Deze wordt bij de request opgenomen in request header "X-Api-Key". Wanneer je je aanmeldt voor het gebruiken van de API ontvang je de API-key.
 
-De url van de testomgeving is https://api.acceptatie.kadaster.nl/lvwoz/api/v1.
+### Testgevallen
+Onderstaande tabellen bevatten testgevallen voor specifieke situaties waarmee de werking van de API kan worden getest op de test omgeving.
```

Comment:
de testgevallen hieronder zie je alleen met de api key die wij hebben. een gemeente of leverancier ziet die testgevallen niet. zij moeten zelf testgevallen maken. het zou toch fantastisch zijn als hier beschreven werd hoe dat moet. (ik heb zelf namelijk geen idee hoe dat moet en wat je daarvoor nodig hebt)

Reply:
maar onderstaande tabel moeten we m.i. niet opnemen

Reply:
Eens